packages/oidc-middleware/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# 2.0.0
+
+### Features
+
+- Adds Okta logout capability (informing Okta that the session is ended rather than just locally forgetting the current session) ([#162](https://github.com/okta/okta-oidc-js/issues/162))
```

Comment:
Please update to use this format https://oktawiki.atlassian.net/wiki/spaces/eng/pages/238007322/Localization+Dev#Localization(Dev)-Inokta-signin-widget

Reply:
Yep sorry I pasted the wrong link, what I meant is to try to keep the format consistent, like: https://github.com/okta/okta-oidc-js/pull/383/files#diff-7edd5b3280c5873d3fea769a1e4de551R26